sql/src/main/java/io/crate/analyze/OrderBy.java

```diff
@@ -195,8 +195,13 @@ public int hashCode() {
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder("OrderBy{");
-        for (int i = 0; i < orderBySymbols.size(); i++) {
-            Symbol symbol = orderBySymbols.get(i);
+        prettyPrint(sb, orderBySymbols, reverseFlags, nullsFirst);
+        return sb.toString();
+    }
+
+    public static StringBuilder prettyPrint(StringBuilder sb, List<Symbol> symbols, boolean[] reverseFlags, Boolean[] nullsFirst) {
```

Comment:
This is generally an issue with the mapRepresentation implementations that I have.
What about adding `toString()` as abstract method to the Projection class to force it to be implemented and add a javadoc to indicate that this is used for EXPLAIN. 

From what we currently use in toString and what this PR tries to achieve I think it's desireable to keep debug information and explain in sync.